lib/acpi/Kconfig

```diff
@@ -58,3 +58,15 @@ config ACPI_HID_LEN_MAX
 	default 12
 	help
 	  Size of HID string.
+
+config ACPI_IRQ_VECTOR_MAX
```

Comment:
shouldn't we be able to use kernel dynamic allocation, so these 2 config would not be necessary?